dags/compute_acteurs/tasks/business_logic/compute_acteur.py

```diff
@@ -60,6 +65,19 @@ def compute_acteur(df_acteur: pd.DataFrame, df_revisionacteur: pd.DataFrame):
         lambda x: shortuuid.uuid(name=x)
     )
 
+    # Get all charfield of Acteur model
+    charfields = {
```

Comment:
```suggestion
    # Get all string fields of Acteur model
    string_fields = {
```

proposition, on récupère les charfield et textfield